app/qml/project/MMProjectWizardPage.qml

```diff
@@ -46,102 +44,100 @@ Item {
     Component.onCompleted: fieldsModel.initModel()
   }
 
-  // background
-  Rectangle {
-    width: parent.width
-    height: parent.height
+  pageHeader {
+    title: qsTr("Create Project")
     color: __style.lightGreenColor
+
+    onBackClicked: root.backClicked()
   }
 
-  MMPageHeader {
-    id: header
-    width: projectWizardPanel.width
-    color: __style.lightGreenColor
-    title: qsTr("Create Project")
 
-    onBackClicked: {
-      projectWizardPanel.backClicked()
-    }
-  }
+  pageContent: Item {
 
-  Item {
-    height: projectWizardPanel.height - header.height - toolbar.height
-    width: projectWizardPanel.width
-    y: header.height
+    width: parent.width
+    height: parent.height
 
-    ColumnLayout {
+    Column {
       id: contentLayout
-      spacing: 10 * __dp
-      anchors.fill: parent
-      anchors.leftMargin: __style.pageMargins
-      anchors.rightMargin: __style.pageMargins
+
+      width: parent.width
+      height: parent.height
+
+      spacing: 0
+
+      MMListSpacer { height: __style.margin20 }
 
       MMTextInput {
         id: projectNameField
+
         title: qsTr("Project name")
-        height: projectWizardPanel.rowheight
         width: parent.width
-        Layout.fillWidth: true
-        Layout.preferredHeight: projectWizardPanel.rowHeight
       }
 
-      Label {
+      MMListSpacer { height: __style.margin20 }
+
+      MMText {
         id: attributesLabel
-        height: projectWizardPanel.rowheight
+
+        height: root.rowheight
         width: parent.width
         text: qsTr("Fields")
         color: __style.nightColor
         font: __style.p6
-        Layout.preferredHeight: projectWizardPanel.rowHeight
         horizontalAlignment: Text.AlignLeft
         verticalAlignment: Text.AlignVCenter
       }
 
       ListView {
         id: fieldList
+
         model: fieldsModel
         width: parent.width
-        Layout.fillWidth: true
-        Layout.fillHeight: true
+        height: parent.height - projectNameField.height - projectNameField.height
```

Comment:
I think you are missing here one spacing that you applied via `MMListSpacer { height: __style.margin20 }` on line 77